File: ws/fuentes/U1/WS10CONSTRUCTORS/ChickenFarmSimulator/src/ec/edu/espe/chickenfarm/view/ChickenFarmSimulator.java
package ec.edu.espe.chickenfarm.view;

import ec.edu.espe.chickenfarm.model.Chicken;
import java.util.Date;

/**
 *
 * @author Jorge Fuentes
 */
public class ChickenFarmSimulator {
    public static void main(String[]args){
        int id=1;
        String color="White and brown";
        String name="Lucy";
        int age=5;
        boolean molting=true;
        Date date=new Date();
        
        Chicken chicken=new Chicken(id, name, color, age, molting, date);
        
        System.out.println("Jorge´s chicken data -->" + chicken);
        System.out.println("Jorge´s chicken data -->");
        chicken.doStuff();
                
        System.out.println("chicken --> " + chicken.getName() + " is " + chicken.getColor());
        
        chicken.setAge(12);
        System.out.println("Chicken --> " + chicken.getName() + " is " + chicken.getAge() + "Year-old");
        System.out.println("Jorge´s chicken data --> " + chicken);
    }
    
}

File: ws/jacome/U1/WS10Constructors/JS version/ChickenFarmSimulator.js
import { Chicken } from './Chicken.js';

function main() {
    const id = 1;
    const name = "Lucy";
    const color = "white and brown";
    const age = 5;
    const molting = true;
    const bornOnDate = new Date();

    const chicken = new Chicken(id, name, color, age, molting, bornOnDate);

    console.log("Micaela’s chicken data -->", chicken.toString());
    console.log("Micaela’s chicken data -->");
    chicken.doStuff();

    console.log(`chicken --> ${chicken.name} is ${chicken.color}`);

    chicken.age = 12;
    console.log(`chicken --> ${chicken.name} is ${chicken.age} year-old`);
    console.log("Micaela’s chicken data -->", chicken.toString());
}

main();
